prisma/language-tools
# feat(LS): references

To test this locally:
1. `npm i && npm run bootstrap` in the root of language-tools
~2. `./prisma-schema-wasm/scripts/update-schema-wasm.sh` in the root of engines~
2. `npm run compile` in the root of language-tools
3. In VS Code open run & debug (cmd + shift + d on mac) and hit `Launch VS Code extension`
---
3.5 if you see the the following error message:
```
Error: Cannot find module '.../language-tools/packages/language-server/dist/bin'
    at Function._resolveFilename (node:internal/modules/cjs/loader:1084:15)
    at node:internal/modules/cjs/loader:929:27
    at Function._load (node:electron/js2c/node_init:2:13672)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:101:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
4. `cd packages/language-server`
5. `tsc -p ./` and re-try launching the extension

companion pr to https://github.com/prisma/prisma-engines/pull/4934
